src/components/SegmentedControl/SegmentedControlOption.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { ReactNode } from 'react';
```

Comment:
We have some inconsistency with file names sometimes are in capital case and others in lower, I think we should use lower case for all files and only component folders will start with a capital case
Also, I think we can name this file simply option and the component Option

Reply:
Agree! Working on the reviews now!